platform/vs/docker-ptf-sai.mk

```diff
@@ -3,7 +3,11 @@
 DOCKER_PTF_SAI = docker-ptf-sai.gz
 DOCKER_PTF_BASE = docker-ptf.gz
 $(DOCKER_PTF_SAI)_PATH = $(DOCKERS_PATH)/docker-ptf-sai
-$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_0_14_1) $(PYTHON3_THRIFT_0_14_1)
+ifeq ($(SONIC_PTF_ENV_PY_VER), mixed)
+$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF) $(PYTHON_SAITHRIFT)
+else
+$(DOCKER_PTF_SAI)_DEPENDS += $(PYTHON_SAITHRIFT)
```

Comment:
Why is Python thrift being removed? This is needed for saithrift to work.

Edit: I see that `python3-thrift` is installed into the Docker container. That should be sufficient.